packages/synapse-react-client/src/theme/DefaultTheme.tsx

```diff
@@ -97,7 +97,7 @@ export const defaultMuiThemeOptions: ThemeOptions = {
             fontWeight: 900,
             padding: padding,
             borderRadius: '3px',
-            textTransform: 'capitalize',
+            textTransform: 'var(--synapse-button-text-transform)',
```

Comment:
Use a CSS variable to avoid the complexity of merging themes.

[MUI has adopted CSS variables](https://mui.com/material-ui/customization/css-theme-variables/overview/) in newer versions (v6, v7), which we could adopt to replace our Sass variables and simplify portal theming.